src/app/contextBroker/contextBroker.cpp

```diff
@@ -1485,8 +1513,6 @@ int main(int argC, char* argV[])
     LM_X(1, ("dbName too long (max %d characters)", DB_NAME_MAX_LEN));
   }
 
-  LM_I(("Orion Context Broker is running"));
```

Comment:
This might be a mistake. There should be ONE

    LM_I(("Orion Context Broker is running"));

    in the code (I copied this line and added a whole bunch for troubleshooting. At cleanup I must have removed them all, including the original one).  Good catch!

I will make sure there is only one.